images/entrypoint.sh

```diff
@@ -35,7 +35,7 @@ if [ "$2" == "origin" ]; then
     export PATH="${ST_HOME}/bin:${QDL_HOME}/bin:${PATH}"
 
     # Run the boot to inject the template
-    ${QDL_HOME}/var/scripts/boot.qdl
+    ${QDL_HOME}/var/scripts/boot.qdl > /dev/null
```

Comment:
Can you add a comment explaining why we do this? Something as simple as "The output from this script is mostly unhelpful to users, so hide it by piping to /dev/null" should be okay.